Data Structure/Linked List/LinkedListHomework2.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+class Node {
+    /** @var int */
+    private $data;
+    /** @var Node */ 
+    private $next; 
+
+    private $prev;
+    /**
+    * Constructor Node class
+    */
+    public function __construct($data = 0, $next = null, $prev = null)  // default value of $data is 0, $next is null
+    {
+        $this->data = $data; // initial $data
+        $this->next = $next;  // initial $next
```

Comment:
update to
`$this->next = $next; // initial $next`

Reply:
Sorry I couldn't understand the meaning of this section. What is the matter of my sentence?

Reply:
ahhh, sorry for my poor explain
your coding has 2 spaces before // .... i'd like to update to 1 space
`$this->next = $next;  // initial $next`

Reply:
Oh! Sorry now I understood. Thank you so much!